scene/resources/resource_format_text.cpp

```diff
@@ -1013,7 +1009,7 @@ Error ResourceLoaderText::save_as_binary(FileAccess *p_f, const String &p_path)
 	wf->store_32(0); //64 bits file, false for now
 	wf->store_32(VERSION_MAJOR);
 	wf->store_32(VERSION_MINOR);
-	static const int save_format_version = 3; //use format version 3 for saving
+	static const int save_format_version = 4; //use format version 4 for saving
```

Comment:
It would be good to expose FORMAT_VERSION somewhere as a public enum in ResourceFormtBinaryLoader/Saver and use it from here.